wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,207 @@
+*Main article: [Beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor)*
 
-**Insert Break Time** button will put/stop "Break Time" in current timestamp. Beside the button is the **short sound file name storyboarded to the selected(blue) note**, known as **keysound** (refer to "Sampling" section at the end of this page).
+**Time signature** (MM:B) from the timing section is added (15:3 = 15th measure 3rd beat) beside the adjustment bar for easier mapper's reference. It can go to negative value (e.g. -8:-3) if the first timing section is a *few seconds away* from the start of the beatmap's song file.
 
-### Centre-Left (Selector)
+Beside the `Insert Break Time` button is the **sound file name storyboarded to the selected note**, known as **keysound**. *Refer to the [Sampling](#sampling) section*
 
-**Your construction toolkit osu!mania ver., made for DIY.**
+### Notes selector for osu!mania
 
-**Please note that in osu!mania's default notes hitsound works different with Standard/osu!**
+![](img/tools-mania.jpg "Notes selector for osu!mania")
 
-- Adding clap on a notes will only play `hitclap` sound instead of `hitclap` + `hitnormal` sound on hit while in osu!mania :
-
-![Hit Notes Selector (osu!mania ver.)](img/BEM_HO.jpg "Hit Notes Selector (osu!mania ver.)")
-
-#### Selector
-
-| Buttons / Header (Keyboard Shortcut) | Usage | Description |
+| Buttons (Keyboard shortcut) | Usage | Description |
 | :-- | :-- | :-- |
-| Sampleset | Auto, Normal, Soft, Drum | Change the Sampleset ignoring the current one on timing section. Ex : Timing **Drum 66%**, while this Sampleset is **Soft**, will play all related **Soft** sound on the notes like clap , normal , finish, whistle with 66% volume. |
-| Additions | Auto, Normal, Soft, Drum | Basically the same as above, BUT Additions setting overwrite Sampleset setting. Ex : Timing **Drum 55%**, Sampleset **Soft**, while this Additions is **Normal**, will play all related **Normal** sound on the notes like clap , normal , finish, whistle with 55% volume. |
-| Select (`1`) | **Left-click/drag:** Move timestamp and location of a note. **Right-Click:** Remove note. **`Ctrl` + Click:** Multiple selection. | Select and modify existing notes. |
-| Circle (`2`) | **Left/Right-click:** Place/Remove note. | Add note to current column based on mouse position. |
-| Hold (`3`) | **Left-click (Hold):** Place hold note and adjust length. **Release on end:** Set hold length | Add hold note to current column based on mouse position. |
-
-### Centre (Playfield)
+| `Sampleset` | Auto, Normal, Soft, Drum | Change the sampleset ignoring the current one on timing section. |
+| `Additions` | Auto, Normal, Soft, Drum | Same as above, but overrides the `Sampleset` setting. |
```

Comment:
this needs to mirror the same rows from the description of the regular editor mode